programs/ssl/ssl_client2.c

```diff
@@ -390,6 +398,40 @@ int main( void )
 #define USAGE_SERIALIZATION ""
 #endif
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
```

Comment:
The existing code includes the protocol version identifiers regardless of whether the respective version is enabled in the config. I'd suggest to continue that pattern and unconditionally add `tls1_3` and `dtls1_3` for the documentation of possible options.